book/book.tex

```diff
@@ -64,7 +69,8 @@
 \include{chapters/hirschvogel/main} 
 \include{chapters/munthe/main}
 \include{chapters/paratico/main}
-\include{chapters/marcibal/main}
+\include{chapters/parsons/main}
+%\include{chapters/marcibal/main}
```

Comment:
We should add back marcibal's chapters

Reply:
Maybe my ref definitions were interfering. Marcibal's chapter is compiling now.  Marcibàl's $\'{a}$  needs to be accented, or utf-8 activated.